login/config.h

```diff
@@ -53,6 +53,14 @@ typedef struct glome_login_config {
   const char* host_id;
 } glome_login_config_t;
 
+#define GLOME_LOGIN_PUBLIC_KEY_ID "glome-v1"
```

Comment:
I think it'd be slightly better to forward declare that here and initialize it as a`static const char GLOME_LOGIN_PUBLIC_KEY_ID[] = "glome-v1";` in the compilation unit (deduplicating multiple instances of the string).

Reply:
Or maybe that's a judgement call as strlen(GLOME_LOGIN_PUBLIC_KEY_ID) will be optimized out anyway at -O2. So I'm just going to approve this as-is.

Reply:
Thanks for the suggestion, but we'd need to expose it for the cli as well, and the string literals should be collapsed into one constant by the compiler, so I'm going to keep it that way.